data/json/items/gun/9mm.json

```diff
@@ -62,6 +62,7 @@
       [ "9mm", [ "calicomag" ] ],
       [ "22", [ "a180mag" ] ],
       [ "45", [ "thompson_drum" ] ],
+      [ "46", [ "hk46bigmag" ] ],
```

Comment:
Same here, wrong shape magazine (if your bothered about reality). The balance concern is that `46` is already better than `57` and limiting conversions of the former can help to offset that